### ISSUE TITLE
Unable to create an order from the admin panel of Magento

### ISSUE THREAD

**Issue by Mayur-Wadpalliwar** [razorpay/razorpay-magento#70]
Steps to re-produce:

Login to Magento admin panel,
- Go to Sales/Order and create new order

Error:

There was an error capturing the transaction: Warning: hash_equals(): Expected user_string to be a string, null given in /var/www/charmsday/vendor/razorpay/magento/Model/PaymentMethod.php on line 272.

**Comment by Mayur-Wadpalliwar** [comment]
Ref: https://razorpay.zendesk.com/agent/tickets/84588

**Comment by mayankamencherla** [comment]
https://github.com/razorpay/razorpay-magento/pull/75

**Comment by mayankamencherla** [comment]
@ccrims0n Can you please share an update on this? I think we can merge the code.

**Comment by pandurangpatil** [comment]
WE are also facing this issue.

**Comment by suribhakar** [comment]
Yes , I am also facing this issue till now, razorpay version(3.6.2).

any can help for this?
